pcdsdevices/utils.py

```diff
@@ -461,6 +464,92 @@ def exclude(
         return set(cls.__members__.values()) - cls.include(identifiers)
 
 
+def set_many(
+    to_set: Dict[ophyd.Signal, OphydDataType],
+    *,
+    owner: Optional[ophyd.ophydobj.OphydObject] = None,
+    timeout: Optional[Number] = None,
+    settle_time: Optional[Number] = None,
+    raise_on_set_failure: bool = False
+) -> ophyd.status.StatusBase:
+    """
+    Call ``set`` on all given signal-to-value pairs with a single Status
+    return value.
+
+    Returns
+    -------
+    status : ophyd.Status.StatusBase
+        One Status or AndStatus instance that reflects the completion status of
+        the setting all signal to the provided values.
+
+    owner : OphydObject, optional
+        The owner object, to be used for logging / Status object attribution.
+
+    timeout : float, optional
+        Per-signal timeout to configure during set.
+
+    settle_time : float, optional
+        Per-signal settle time to configure during set.
+
+    raise_on_set_failure : bool, optional
+        Raise if any of the ``set`` calls fail.
```

Comment:
This docstring is malformed, the parameters section has lost its header and dropped below the returns

Reply:
Good catch, and weird